README.md

```diff
@@ -5,4 +5,67 @@
 
 Corruption is a huge bane to Africa’s development. African countries must develop novel and localised solutions that will curb this menace, hence the birth of iReporter. iReporter enables any/every citizen to bring any form of corruption to the notice of appropriate authorities and the general public. Users can also report on things that needs government intervention
 
+## Getting started
+You can clone this [repo](https://github.com/Emmanuelaaron/iReporter.git) on your local machine or checkout the user-interface on [gh-pages](https://emmanuelaaron.github.io/iReporter/UI/temps/start_page) and the app is hoted on [heroku](https://ireportere.herokuapp.com/)
 
+### Prerequisites
+Install [python](https://www.python.org/downloads/release/python-371/) on your local machine
+
+### Installing
+Clone the this repo on your local machine
+```
+$ git clone https://github.com/Emmanuelaaron/iReporter.git
+```
+cd into the cloned directory, install the virtual environment and activate, checkout to the most stable branch and install all the dependences.
+```
+$ cd iReporter
+$ pip install virtualenv
+$ virtualenv venv
+$ source venv/Scripts/activate
+$ git checkout api-endpoints
+$ pip install -r requirements.txt
+$ python run.py
+```
+* copy the Url it into postman and put to run any endpoint of your preference in the table below with the url prefix ('/api/v1') for each endpoint.
+
+HTTP Method | Endpoint | Functionality | Parameters 
+------------|----------|---------------|------------
+POST | /signup | User able to signup | None
+POST | /red-flags | Creates a red flag| None
+GET | /red-flags | Gets all red flags | None
+GET | /red-flags/<int:flag_id> | Gets a specific red flag | Flag_id
+DELETE | /red-flags/<int:flag_id> | deletes a specific red flag | Flag_id
+
+## Running Tests
+Install pytest, activate the virtual environment and then run the tests
+```
+$ pip install pytest
```

Comment:
since you already installed requirements, this line is redundant and already satisfied.

Reply:
Okay then let me remove it